index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div class="container columns">
+      <nav id="navbar" class="column is-2">
+        <header>Navigation</header>
+        <a href="#JavaScript_and_Java" class="nav-link">JavaScript and Java</a>
```

Comment:
Wonderful work being polite in your review, but I think you can do better by explaining why they should put the correct value in the `href` attribute of any `a` tag. For example, it would be nice if your statement could be like this _**"Please fix your a tag's href elements of navbar. They are currently placed in error and when you click on the element, it would not direct you to the right part of the website."**_